scrunch/datasets.py

```diff
@@ -2425,9 +2425,15 @@ def fork(self, description=None, name=None, is_published=False,
         else:
             if project:
                 # Create fork in given Project path.
-                body["project"] = (
-                    project if project.startswith("http") else get_project(project).url
-                )
+                try:
+                    project = get_project(project).url
```

Comment:
```suggestion
                    project_url = get_project(project).url
```
It will help to not overwrite the variable and also naming it `project_url` is more clear that this is storing a URL (string) and not a `Project()` instance. Ideally we'd do `project_url:str = ....` but python 2 😒

Reply:
Agreed! 😃